cognite/client/_api/relationships.py

```diff
@@ -311,6 +310,48 @@ def create(self, relationship: Union[Relationship, List[Relationship]]) -> Union
 
         return self._create_multiple(items=relationship)
 
+    def update(
+        self, item: Union[Relationship, RelationshipUpdate, List[Union[Relationship, RelationshipUpdate]]]
```

Comment:
might as well add an example that takes in `Relationship` instead of `RelationshipUpdate`.